packages/email/email.js

```diff
@@ -145,17 +206,24 @@ Email.hookSend = function (f) {
  * `new EmailInternals.NpmModules.mailcomposer.module`.
  */
 Email.send = function (options) {
-  for (var i = 0; i < sendHooks.length; i++)
-    if (! sendHooks[i](options))
-      return;
-
   if (options.mailComposer) {
     options = options.mailComposer.mail;
   }
 
-  var transport = getTransport();
+  let send = true;
+  sendHooks.each(hook => {
+    send = hook(options);
+    return send;
+  });
+  if (!send) return;
+
+  const customTransport = Email.customTransport;
+  const transport = customTransport ? false : getTransport();
   if (transport) {
     smtpSend(transport, options);
+  } else if (customTransport) {
+    const packageSettings = Meteor.settings.packages?.email || {};
+    customTransport({ settings: { ...packageSettings }, ...options, });
```

Comment:
I would call settings as packageSettings as settings would be confusing with Meteor settings.

Reply:
Got it! Fixed! Now fixing the docs...